Update Next/Nuxt guides (#746)
edgio-docs/edgio-docs: src/templates/origins_config_warning.md

<Callout type="warning">
  

>   Using the root `origins` configuration will apply to all environments and potentially overwrite origin configurations when your property is deployed to {{ PRODUCT }}.

Reviewer: ```suggestion
  You may define the `origins` configuration at the root or within an environment-specific object (e.g., `environments.production.origins[0]`). Defining it at the root may overwrite your origin configurations for the environment to which your build is being deployed. If your environments use different origin configurations, we recommend that you define it within an environment-specific object.
```

>   See [Environments](/guides/basics/environments) for more information on configuring multiple environemnts and origins for your property.

Reviewer: ```suggestion
  Learn more about [environments](/guides/basics/environments) and the [{{ CONFIG_FILE }} file](/guides/performance/cdn_as_code/edgio_config).
```
